test/NHSD.BuyingCatalogue.Documents.API.Tests/NHSD.BuyingCatalogue.Documents.API.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="fluentassertions" Version="5.10.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8">
```

Comment:
Do we need FxCopAnalyzers for test code?

Reply:
@sam-roberts-bjss good question. While it's obviously not as important as non-test code, the test code is part of the code base so I'd argue that it's still worthwhile applying best practices (although the rule set applied will almost certainly more relaxed than that used for the non-test code).